src/literals.py

```diff
@@ -10,3 +10,122 @@
 REL_NAME = "kafka-client"
 CHARM_USERS = ["sync"]
 TLS_RELATION = "certificates"
+
+EXPORTER = """lowercaseOutputName: true
```

Comment:
Uhm, what if we rather create in the repo a file (confs/jmx-exporter.yaml) that is read and streamed (or just copied) to the workload specific location? 

I believe we should improve in readibility

Reply:
I really like the idea of templating. But, does this requires some relevant extra work hard to be included in this PR or can we address @marcoppenheimer comment within the PR?

If we cannot, I would at least move this to a dedicated file within the PR (this should be little work) and place an item in the backlog.

Reply:
Yeah, this can be easily changed. I'll move this to a different place where it makes more sense.